### DISCUSSION TITLE
Importante!! Agreguen su correo de Innovaccion a outlook

### DISCUSSION THREAD

**Discussion by YomairaBarrera** [LaunchX-InnovaccionVirtual/CursoIntroPython#171] [General]
Buenas noches compañerxs, nada mas un recordatorio para quien no lo haya hecho, abran su correo de Innovaccion en el Outlook para que puedan visualizar sus correos después de que envíen el form el lunes, porque acuérdense que van a mandar los correos al email que nos dieron iniciando el curso.

Saludos

**Comment by LuisSantiago10** [comment]
Gracias por la información

**Comment by AlexZRamirez** [comment]
Muchas gracias por la información

**Comment by AdanGarmendia** [comment]
Gracias por el recordatorio 🤝🏼

**Comment by OscarIslasH** [comment]
Tengo una duda, al tratar de entrar a mi cuenta de correo de inovacion, tengo problemas para iniciar sesion, esto solo me ocurre en la aplicacion de correo de windows 11 y en MicrosoftTeams. Si me deja iniciar sesion desde el navegador pero no desde las apps de windows, si alguien pudiera ayudarme con esto se los agradeceria.

Pd: No estoy escribiendo mal el correo, es el mismo que me sale en teams desde el navegador al cual si me deja entrar.
![Captura de pantalla 2022-02-12 043414](https://user-images.githubusercontent.com/99118114/153707978-3d740a93-205a-4d99-a611-23e20fb799fa.png)
![Captura de pantalla 2022-02-12 043922](https://user-images.githubusercontent.com/99118114/153708009-4494b558-d69b-4740-9899-62d5d4b9e272.png)

.

**Reply by YomairaBarrera** [reply]
Creo que precisamente tiene que ver con el mensaje de advertencia que te sale al final tienes el Microsoft teams para uso personal es un caso particular del Windows 11 chécate si no me equivoco en la primer sesión hablan sobre eso. Y no esta de mas que les mandes un mensaje al insta de innovaccion a ver si te pueden asesorar

**Reply by maikcore** [reply]
Hola para mi que estás usando la versión normal de TEAMS descarga la versión de TEAMS para el trabajo. https://go.microsoft.com/fwlink/p/?LinkID=869426&clcid=0x80a&culture=es-mx&country=MX&lm=deeplink&lmsrc=groupChatMarketingPageWeb&cmpid=directDownloadWin64
